core/src/avm1/activation.rs

```diff
@@ -2834,7 +2809,6 @@ impl<'a, 'gc, 'gc_context> Activation<'a, 'gc, 'gc_context> {
             self.set_variable(name, value)
         } else {
             let scope = self.scope;
```

Comment:
nit: Let's line this up to `self.scope.define_local(...)`

Reply:
I did this at the start, but I ended up with weird borrowck errors; I could try changing it to `self.scope()...` though.

Reply:
Fixed!